app/services/location_lookup_service.rb

```diff
@@ -49,7 +49,7 @@ def areas
 
   def response
     @response ||= begin
-                    JSON.parse(mapit.location_for_postcode(postcode).to_json)
+                    Rails.cache.fetch(JSON.parse(mapit.location_for_postcode(postcode).to_json), expires_in: 3.hours)
```

Comment:
This doesn't look like correct usage of the API. As I understand the usage is along the lines of:

`Rails.cache.fetch(key_name, expires_in: 3.hours) { code_to_run_to_populate_cache }`

This should provide an example: https://guides.rubyonrails.org/caching_with_rails.html#low-level-caching

Reply:
ah ace yeah, sorry we're just figuring out the API - preaturely tagged you in this!

Reply:
sorry swaped that over now, we're using the postcode as the key, that seemed like a good idea?